notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_MTM1M3_Position_Stability_17Nov23.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>I think that the <code>Spec limits</code> legend is misplaced . I guess that it should be moved to the first plot just after the "Now make the large elevation slew plot" cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>